doc/src/tutorials/physics/index.rst

```diff
@@ -22,3 +22,4 @@ write to our `mailing list
    :maxdepth: 2
 
    biomechanics/index.rst
+   duffing-example.rst
```

Comment:
This should be under `mechanics/duffing-example.rst`.

Reply:
We should also create an `mechanics/index.rst` file, but this can be addressed when solving #25607